### ISSUE TITLE
✨ Feature Suggestion | I have checked the about:config recommendations of Firefox in your website and I am pretty impressed. I want to give an advice of adding three more config settings and I will give description of them below.

### ISSUE THREAD

**Issue by Lostname777** [privacytoolsIO/privacytools.io#1834]
1) geo.enabled = false
Description: Setting this to false will disable geolocation tracking, which may be requested by a site you are visiting. [As explained by Mozilla](https://support.mozilla.org/en-US/kb/does-firefox-share-my-location-websites?redirectlocale=en-US&redirectslug=does-firefox-share-my-location-web-sites), this preference is enabled by default and utilizes Google Location Services to pinpoint your location. In order to do that, Firefox sends Google:

   1. your computer’s IP address
   2. information about nearby wireless access points
   3. a random client identifier, which is assigned by Google (expires every two weeks)

Before this data is sent to Google, you would first get a request by the site you are visiting. Therefore you do have control over this, even if geo remains enabled.

2) network.cookie.lifetimePolicy = 2
Description: This is another integer type preference that you should set to a value of 2. This preference determines when cookies are deleted. Here are the different options:

    0 = Accept cookies normally
    1 = Prompt for each cookie
    2 = Accept for current session only
    3 = Accept for N days

With a value of 2, websites you visit should work without any problems, and all cookies will be automatically deleted at the end of the session.

3) network.cookie.cookieBehavior
Description: This is an integer type preference with different values. Here are the cookie preference options:

    0 = Accept all cookies by default
    1 = Only accept from the originating site (block third-party cookies)
    2 = Block all cookies by default
    3 = Block cookies from unvisited sites
    4 = New Cookie Jar policy (prevent storage access to trackers)

Any selection between 1 and 4 would improve privacy. The New Cookie Jar policy (value 4) offers more protection, but it may also break the functionality of some websites. Ghacks has a discussion of the New Cookie Jar policy [here](https://www.ghacks.net/2018/09/23/firefox-65-new-cookie-jar-policy-to-block-tracking/).

**Comment by Thorin-Oakenpants** [comment]
tl;dr: your data is factually incorrect, out of date, and NO, NO and NO

---

Se #1212 and #1430 

1. NO
  - geo is already behind a prompt
  - users can manage those site permissions from the urlbar, and from options
  - so all you're doing is adding yet another barrier to functionality
  - and while you're at it: congrats: your fingerprint changed (geo default permissions can be read by JS)

2. NO
   - your info is woefully out of date
   - for N days does not exist: it was removed in FF63, and it relies on another pref (network.cookie.lifetime.days - which was also removed at the same time) - here the [proof](https://bugzilla.mozilla.org/show_bug.cgi?id=1457170)
   - prompt for each cookie does not exist - I'm not going to track down the bugzilla
   - accept for session only
      - using session only cookies can cause breakage
      - session cookies don't have the same privileges as normal cookies: for example shared and service workers will fail
      - it depends exactly how Mozilla would have implemented this but I would expect shit to break
   - there's a better and cleaner solution: just clear cookies on close

3. NO
   - it's already on the PTIO page
   - the idea is to get rid of it from the list and make users use the UI
   - the default value since FF69 is not 0, it is 4
   - why are you pointing at a 19 month old article and using an old name for value 4: this is not how it works anymore: it's been constantly evolving, even today (e.g. they just added dynamic FPI to it) and is used for ETP
   - this has been discussed recently (but I can't find the issue), and there's no need to include confusing settings e.g
      - option 3 is a waste of time, option 4 is confusing as fuck for people (see last point re ETP)
   - "The New Cookie Jar policy (value 4) offers more protection"
      - bollocks: blocking all cookies offers more protection: you need to quantify the argument
